3.1版本修改 (#1787)
cocos-creator/creator-docs: en/editor/scene/index.md

## Scene settings

> **Scene Editor** The options in the top right corner can be used to set the scene reference image, scene grid, scene camera parameters, etc. Please refer to the description below for details.

Reviewer: ```suggestion
The options in the top right corner of the **Scene** panel can be used to set the reference image, grid, camera parameters, etc. of the scene. Please refer to the description below for details.
```

### Reference image settings

> Click on the ! [scene reference](images/scene-reference.png) button in the top right corner of the **Scene Editor** to open the **Reference Diagram** panel, which is primarily used as a cross-reference for developers when stitching together UI in the **Scene Editor**.

Reviewer: ```suggestion
Click on the ![scene reference](images/scene-reference.png) button in the top right corner of the **Scene** panel to open the **Reference Image** panel, which is primarily used as a cross-reference for developers when stitching together UI in the **Scene** panel.
```

![reference](images/reference.png)

| Option | Description |
| :-- | :-- |

> | Add |For adding UI reference images, multiple reference images can be added.  |

Reviewer: ```suggestion
| Add |Used to adding UI reference images. Multiple reference images can be added.  |
```

| Delete | Used to remove the UI reference image from the currently selected box. |

> | X | The X-axis position of the selected UI reference image is displayed in the scene. |

Reviewer: ```suggestion
| X | The X-axis position of the selected UI reference image in the scene. |
```

> | Y | The selected UI reference image is displayed on the Y-axis in the scene |

Reviewer: ```suggestion
| Y | The Y-axis position of the selected UI reference image in the scene. |
```

> | Opacity | Opacity of the selected UI reference image displayed in the scene |

Reviewer: ```suggestion
| Opacity | Opacity of the selected UI reference image displayed in the scene. |
```
